Templates/Swift/InterfaceWrapper.stencil

```diff
@@ -73,13 +73,13 @@
 		}
 	}
 
-	{% map baseFields into initArgs using field %}{{ field.name }}: {{ field|renderType }}{% endmap %}
+	{% map baseFields into initArgs using field %}{{ field.name|escapeReservedWord }}: {{ field|renderType }}{% endmap %}
 	{% map fragmentSpreads into initFragmentArgs using fragment %}{{ fragment.name|lowercasedFirstLetter }}Fragment: {{ moduleName }}.{{ fragment.name }}{% if fragment.conditionallySelected %}?{% endif %}{% endmap %}
 
 	{{ accessLevel }} init(__typename: String, realized: Realized{% if initArgs %}, {% endif %}{{ initArgs|join:", " }}{% if initFragmentArgs %}, {% endif %}{{ initFragmentArgs|join:", " }}) {
 		self.__typename = __typename
 		self.realized = realized
-		{% map baseFields into commonArgs using field %}{{ field.name }}: {{ field.name }}{% endmap %}
+		{% map baseFields into commonArgs using field %}{{ field.name }}: {{ field.name|escapeReservedWord }}{% endmap %}
```

Comment:
Didn't escape the first part of this because Swift gives a warning saying escaping an argument when passing a value is not needed